crates/nargo_cli/src/cli/verify_cmd.rs

```diff
@@ -101,10 +101,10 @@ fn verify_with_path<B: Backend, P: AsRef<Path>>(
     // Load public inputs (if any) from `verifier_name`.
     let public_abi = abi.public_abi();
     let (public_inputs_map, return_value) =
-        read_inputs_from_file(program_dir, verifier_name.as_str(), Format::Toml, &public_abi)?;
+        read_inputs_from_file(&package.root_dir, verifier_name, Format::Toml, &public_abi)?;
 
     let public_inputs = public_abi.encode(&public_inputs_map, return_value)?;
-    let proof = load_hex_data(&proof_path)?;
+    let proof = load_hex_data(proof_path)?;
```

Comment:
Similarly, we're reading the same proof for each package

Reply:
We dropped `proof` and are just using package name now.